app/shared/productResponse.ts

```diff
@@ -59,12 +59,23 @@ export interface ProductDetail extends WithSubscription {
   selfServiceCancellation: SelfServiceCancellation;
 }
 
+export interface CancelledProductDetail {
+  mmaCategory: GroupedProductTypeKeys;
+  tier: string;
+  joinDate: string;
+  subscription: CancelledSubscription;
+}
+
 export function isProduct(
   data: MembersDataApiItem | undefined
 ): data is ProductDetail {
   return !!data && data.hasOwnProperty("tier");
 }
 
+export const isCancelledProduct = (
+  data: CancelledProductDetail | ProductDetail | undefined
+) => !!data && !data.hasOwnProperty("joinDate");
```

Comment:
members-data-api should probably return explicit field for this, something like, `isCancelledAndEnded`.

Reply:
^ that would be nice :)